Make install fails

## Problem 

The instructions in README.md to run `make install` to install create-comit-app does not work and logs the following error
```
192-168-1-105:create-comit-app rishflab$ make install
rustup run --install 1.41.1 cargo --color always install --force --path create-comit-app 
error: `/Users/rishflab/create-comit-app/create-comit-app` is not a directory. --path must point to a directory containing a Cargo.toml file.
make: *** [install] Error 101
```

## Goal
Get `make install` working

## Recommendation
This is issue is most likely because the crates are called `create` and `scripts`. The Makefile thinks the names of the crates are `create-comit-app` and `comit-scripts`

--- a/Makefile
+++ b/Makefile
@@ -38,8 +38,8 @@ install_tomlfmt: install_rust
 ## User install
 
 install:
-	$(CARGO) install --force --path create-comit-app $(INSTALL_ARGS)
-	$(CARGO) install --force --path comit-scripts $(INSTALL_ARGS)
+	$(CARGO) install --force --path create $(INSTALL_ARGS)
+	$(CARGO) install --force --path scripts $(INSTALL_ARGS)
 
 clean:
 	$(CARGO) clean

